native/src/modules/endpoint/reducers/pushEvent.js

```diff
@@ -4,7 +4,7 @@ import type { CityContentStateType, EventRouteStateType } from '../../app/StateT
 import type { PushEventActionType } from '../../app/StoreActionType'
 import { EventModel } from 'api-client'
 import ErrorCodes from '../../error/ErrorCodes'
-import { entries, values } from '../../../utils/object'
+import { values, entries } from 'translations'
```

Comment:
Why is this now part of `tranlsations`?

Reply:
I moved it as it is used in both projects. translations is used in every other package in the repo.

I think this is better instead of creating a new utils package